src/util/mpir_pmi.c

```diff
@@ -202,6 +203,7 @@ int MPIR_pmi_init(void)
     MPIR_Process.rank = rank;
     MPIR_Process.size = size;
     MPIR_Process.appnum = appnum;
+    HASH_FNV(pmi_kvs_name, strlen(pmi_kvs_name), MPIR_Process.world_id);
```

Comment:
@hzhou A comment on compatibility with PMIx: `pmi_kvs_name` only exists if `USE_PMI1_API` is defined. I run into a compile error for PMIx here. I think this hash needs a `#ifdef` case distinction for PMI1, PMI2 and PMIx. Or am I overlooking something?

Reply:
It was our negligence. Thanks for pointing it out. I'll push a PR to fix it.